dev/snippets/nav/bar_basic.cljs

```diff
@@ -1,9 +1,11 @@
 #_
 (:require [om-bootstrap.button :as b]
-          [om-bootstrap.nav :as n])
+          [om-bootstrap.nav :as n]
+          [om-tools.dom :as d :include-macros true])
 
 (n/navbar
- {}
+ {:brand (d/a {:href "#"}
```

Comment:
In order for the :collapsible? true to work, we have to specify either a :brand, :toggle-button, or :toggle-nav-key. Otherwise the navbar-header won't render, and the toggle-button won't show up